### ISSUE TITLE
Make https://riot.im/nightly point somewhere similar to https://riot.im/download/desktop/

### ISSUE THREAD

**Issue by turt2live** [vector-im/riot-web#13564]
So we don't have to remember that https://packages.riot.im/nightly/ does a thing then dig out the thing we want.

**Comment by nadonomy** [comment]
Getting a 404 on riot.im/nightly, should it be packages. like the issue description?

**Comment by t3chguy** [comment]
No, that's packages.riot.im/nightly like in the issue description
